### ISSUE TITLE
Subsampling of the training dataset

### ISSUE THREAD

**Issue by LukasBommes** [open-mmlab/mmdetection#8331]
I want to determine how the number of training images affects segmentation mAP. To this end, I would need to take subsets of the training data on which the model is trained. Is there a way to do this out of the box? I couldn't find anything suitable in the train pipeline config.

I know that pytorch provides a [torch.utils.data.Subset](https://pytorch.org/docs/stable/data.html#torch.utils.data.Subset) method that solves this problem. If it is not already the case, could dataset subsampling be included as an option of the training pipeline?

EDIT: I do not want to create separate datasets with different numbers of images, as this would take large amounts of disk space due to the size of my dataset.

**Comment by BIGWangYuDong** [comment]
Sorry for the late reply, for an easy way, is that possible to use different annotations so that you can choose your training and validation set?

**Comment by lengmo1996** [comment]
Is there any update on the problem? I think modifiing annotations for the divided training process are still inconvenient
